test-util/util.js

```diff
@@ -216,10 +216,10 @@ function testRunsAndHasCheckBlocks(it, name, toEval, specs) {
       self.browser.wait(function () {
         return self.browser.isElementPresent(webdriver.By.className("check-results-done-rendering"));
       }, 20000);
-      return response.findElements(webdriver.By.className("check-block-result"));
+      return response.findElements(webdriver.By.className("check-block"));
     });
     var blocksAsSpec = checkBlocks.then(function(cbs) {
-      var tests = cbs.map(function(cb, i) {
+      var tests = cbs.slice(1).map(function(cb, i) {
```

Comment:
For posterity, the slice is removing the div containing "5 passed, 3 failed" in:

```
|-------------------------|
  5 passed 3 failed
|-------------------------|

|-check-block-1-----------|
  line 54 passed
|-------------------------|

|-check-block-2-----------|
  line 56 passed
|-------------------------|
```